src/service/video-manager-impl.js

```diff
@@ -437,7 +437,17 @@ export function supportsAutoplay(win, isLiteViewer) {
     opacity: '0',
   });
 
-  detectionElement.play();
+  try {
+    const playPromise = detectionElement.play();
+    if (playPromise && playPromise.catch) {
```

Comment:
Are there browsers that don't return a promise? As in, they have `#play`, but not the correct implementation.